index.js

```diff
@@ -5,21 +5,41 @@ const path = require('path');
 const Rollup = require('broccoli-rollup');
 const BuildTailwindPlugin = require('./lib/build-tailwind-plugin');
 
+const buildDestinations = {
+  dummy: {
+    path: 'tests/dummy/app',
+    type: 'app'
+  },
+  app: {
+    path: 'app',
+    type: 'app'
+  },
+  addon: {
+    path: 'addon',
+    type: 'addon'
+  }
+};
+
+const validBuildTargets = Object.keys(buildDestinations);
+
 module.exports = {
   name: 'ember-cli-tailwind',
 
   included(includer) {
     this._super.included.apply(this, arguments);
-
-    this.import('vendor/etw.css');
-
-    if (includer.trees) {
-      this.projectType = 'app';
-      this.tailwindInputPath = this._getInputPath(this.project.root, includer.trees.app);
-    } else if (includer.treePaths) {
-      this.projectType = 'addon';
-      this.tailwindInputPath = this._getInputPath(includer.root, includer.treePaths.addon);
+    
+    let buildTarget = includer.options['ember-cli-tailwind']['buildTarget'];
```

Comment:
We should probably guard for `options` or `options['ember-cli-tailwind']` being undefined here

Reply:
👍Fixed [here](https://github.com/embermap/ember-cli-tailwind/pull/11/commits/f309bbfe1f17a1d1eba052ba0a86204934a37f23)